application/modules/home/controllers/Home.php

```diff
@@ -31,6 +31,9 @@ public function index() {
 		// get list slide
 		$this->data["list_slide"] = $this->common_model->get_all_slide(NULL, 'sort_order ASC');
 
+		// get list latest news
+		$this->data["latest_news"] = $this->common_model->get_all_news(NULL, 'created DESC', 3);
```

Comment:
__[code_sniffer]__
* Single-quoted strings should be used unless it contains variables, special chars like \n or single quotes.

<!-- Comment by [SideCI](https://sideci.com) -->